src/js/themes/vanilla.js

```diff
@@ -113,10 +113,7 @@ export default deepFreeze({
       },
     },
     drop: {
-      backgroundColor: {
-        light: '#f8f8f8',
-        dark: '#222222',
-      },
+      backgroundColor: '#f8f8f8',
```

Comment:
Can we change this to just be `background`? I think the way `backgroundStyle` works we can accept color names, HEX colors, opacity, etc.